### PR TITLE
XDR IR fix when close_xdr_incident is False

### PR DESCRIPTION
**Pull request by noydavidi**
Fixed outgoing mirroring issue in XDR IR where incidents were closed even when close_xdr_incident was set to False.

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-43634)

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CortexXDR/ReleaseNotes/6_1_88.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+
+- Fixed an issue with outgoing mirroring where closing an incident in XSOAR with close_xdr_incident set to False resulted in a 400 error.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXDR/ReleaseNotes/6_1_88.md:6]:
```suggestion
- Fixed an issue with outgoing mirroring when closing an incident in Cortex XSOAR with *close_xdr_incident* set to False resulted in a 400 error.
```

**Comment by ShirleyDenkberg** [comment]
@maimorag Doc review completed.

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py [hunk 2]
```diff
@@ -1032,9 +1032,9 @@ def update_remote_system_command(client, args):
 
             close_xdr_incident = argToBoolean(client._params.get("close_xdr_incident", True))
 
-            if not close_xdr_incident:
-                demisto.debug(f"Reverting to previous status {remote_args.data.get('status')}")
-                update_args['status'] = remote_args.data.get('status')
+            if not close_xdr_incident and (update_args.get('status') in XDR_RESOLVED_STATUS_TO_XSOAR):
```
Comment (Shellyber) [inline Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py:1035]:
```suggestion
            if not close_xdr_incident and (update_args.get('status') in XDR_RESOLVED_STATUS_TO_XSOAR):
```
Shouldn't this be: XSOAR_RESOLVED_STATUS_TO_XDR?

**Review by Shellyber — COMMENTED** [review]
Nice one

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py (hunk 2 above)
Comment (noydavidi) [reply Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py]: Fixed

**Comment by Shellyber** [comment]
@noydavidi   Let's add to the test also a case where the status should be passed like 'under_investigation'

**Review by Shellyber — APPROVED** [review]
Nice Job!

**Pull request merged**